projects/cashmere/src/lib/navbar/navbar-mobile-menu/navbar-mobile-menu.component.html

```diff
@@ -1,8 +1,8 @@
-<div class="menu-dropdown" [style.top.%]="_yPos">
-    <ng-content></ng-content>
-    <div class="bottomSpacer"></div>
-    <a *ngIf="appSwitcher=='true' && appSwitcherService?.allApplicationsUri" class="app-switcher" href={{appSwitcherService?.allApplicationsUri}}>
-        <hc-icon fontSet="fa" fontIcon="fa-th"></hc-icon>
-        <span>View all my applications</span>
-    </a>
-</div>
+<div class="menu-dropdown" [style.top.%]="_yPos">
+    <ng-content></ng-content>
+    <div class="bottomSpacer"></div>
+    <!--<a *ngIf="appSwitcher=='true' && _appSwitcherService?.allApplicationsUri" class="app-switcher" href={{_appSwitcherService?.allApplicationsUri}}>
```

Comment:
I don't know what the plan is with this and I wont say it's a deal breaker but I hate commented out code. It's best to delete and refer to the git history if needed. If this is temp than I'd say it's okay

Reply:
Yeah - Ben and I just weren't able to fix it on the phone this morning.  But you reminded me that I need to reopen that issue.  I'll do that now